frontend/src/App.svelte

```diff
@@ -109,9 +111,14 @@
 
   function handleShortcut(e) {
     // Don't handle keypresses from within text inputs.
+    // Disable shortcuts in views with text inputs, otherwise misclicking outside of a text area may
+    // cause users to accidentally run shortcuts.
     if (
-      ["input", "textarea"].includes(e.target?.tagName.toLocaleLowerCase()) ||
-      e.target.isContentEditable
+      (["input", "textarea"].includes(e.target?.tagName.toLocaleLowerCase()) ||
+        e.target.isContentEditable ||
+        modifierView === ComponentsView ||
+        modifierView === FindReplaceView) &&
```

Comment:
Shortcuts should probably be disabled if any modifier view is set.

```suggestion
        modifierView) &&
```

The linter will probably fail after this suggestion is applied.

Reply:
Given that, I think the entire conditional can be simplified to `if (modifierView)`...

Reply:
I disagree, I think we still need to check for whether we're in `input`, `textarea`, or content editable elements because it is possible for those to exist outside of modifier view panes. For example, we have the jump to offset input that exists in the hex view.